assets/styles/variables/_specials.scss

```diff
@@ -306,6 +306,11 @@ $pullquote-right-width: 45% !default;
 /// @type String
 $pullquote-right-margin-right: 1em !default;
 
+/// Text alignment for right pullquotes on elements with a class of `pullquote-right`.
+/// @type String
+/// @since 1.0.0
+$pullquote-right-align: right !default;
```

Comment:
As per your comment, @dannylonglegs, this should default to center-alignment and then you can override it in the themes (Andreessen and Tufts) where it is right-aligned.